docs/sphinx/source/modelchain.rst

```diff
@@ -232,57 +251,59 @@ plane of array irradiance. The
 assigns default values for temperature (20 C)
 and wind speed (0 m/s) if these inputs are not provided.
 :py:meth:`~pvlib.modelchain.ModelChain.prepare_inputs` requires all irradiance
-components (GHI, DNI, and DHI). See
-:py:meth:`~pvlib.modelchain.ModelChain.complete_irradiance` and
+components (GHI, DNI, and DHI). The
+:py:meth:`~pvlib.modelchain.ModelChain.complete_irradiance`
+method is available for calculating the full set of GHI, DNI, or DHI if
+only two of these three series are provided. See also
 :ref:`dniestmodels` for methods and functions that can help fully define
 the irradiance inputs.
 
 Next, :py:meth:`~pvlib.modelchain.ModelChain.run_model` calls the
 wrapper methods for AOI loss, spectral loss, effective irradiance, cell
 temperature, DC power, AC power, and other losses. These methods are
-assigned to standard names, as described in the next section.
+assigned to generic names, as described in the next section.
 
 The methods called by :py:meth:`~pvlib.modelchain.ModelChain.run_model`
-store their results in a series of ModelChain attributes: ``times``,
-``solar_position``, ``airmass``, ``irradiance``, ``total_irrad``,
-``effective_irradiance``, ``weather``, ``temps``, ``aoi``,
-``aoi_modifier``, ``spectral_modifier``, ``dc``, ``ac``, ``losses``.
+store their results in the ``results`` attribute, which is an instance of
+:py:class:`~.ModelChainResult`. :py:class:`~.ModelChainResult` has the
+following attributes:
+``weather``, ``times``, ``solar_position``, ``airmass``, ``total_irrad``,
+``aoi``, ``aoi_modifier``, ``spectral_modifier``, ``effective_irradiance``,
+``cell_temperature``,  ``dc``, ``ac``, ``losses``, ``tracking``,
+``diode_params``.
```

Comment:
Perhaps just include a couple of these and refer the reader to the class documentation for more. I'm worried we're going to forget to update this comprehensive list in future enhancements.